css/styles.css

```diff
@@ -103,3 +103,24 @@ ul {
 .last-entries-text {
   font-size: 48px;
 }
+
+i {
+  font-size: 25px;
+  padding-left: 20px;
+  padding-top: 10px;
+  padding-right: 10px;
+  padding-bottom: 10px;
+  cursor: pointer;
+}
+
+/* i:hover {
+  color: red;
+} */
```

Comment:
Delete all old code. Although I do think you should keep this. It adds a nice effect. If you keep it, make sure to add a `transition` to the `i` element